drupal/fabfile.py

```diff
@@ -368,12 +377,16 @@ def initial_build_wrapper(url, www_root, repo, branch, build, site, alias, profi
     # any manual clean-up first. Everything else will have run, such as generate drush alias and
     # webserver vhost, so the issue can be fixed and the job re-run.
     if buildtype == "custombranch":
-      FeatureBranches.initial_db_and_config(repo, branch, build, site, import_config, drupal_version)
+      FeatureBranches.initial_db_and_config(repo, branch, build, site, import_config, import_config_method, cimy_mapping, drupal_version)
     else:
       execute(InitialBuild.initial_build_updatedb, repo, branch, build, site, drupal_version)
       execute(Drupal.drush_clear_cache, repo, branch, build, site, drupal_version)
       if import_config:
-        execute(InitialBuild.initial_build_config_import, repo, branch, build, site, drupal_version)
+        if import_config_method == "cimy":
```

Comment:
These code blocks are slightly different, but still feels to me like you could refactor this into the single function I describe above, so you have one place to tweak things if you ever need to change the command or add a method.